Sources/NIO/FileRegion.swift

```diff
@@ -12,6 +12,68 @@
 //
 //===----------------------------------------------------------------------===//
 
+struct _UInt56: ExpressibleByIntegerLiteral {
```

Comment:
Might be nice to consolidate our helper integers into a single source file.

Reply:
fair enough, will do that